Tasks/GoTool/gotool.ts

```diff
@@ -55,7 +49,11 @@ async function acquireGo(version: string): Promise<string> {
         downloadPath = await toolLib.downloadTool(downloadUrl);
     } catch (error) {
         tl.debug(error);
-        throw (tl.loc("FailedToDownload", version, error));
+
+        // cannot localized the string here because to localize we need to set the resource file.
+        // which can be set only once. vsts-task-tool-lib/tool, is already setting it to different file.
+        // So left with no option but to hardcode the string. Other tasks are doing the same.
```

Comment:
If true, this seems like a bug in the tool library. Not all tasks are doing this.

Reply:
Only the task which require tool library, they have hard-coded it. Example dotnetcore, node, Java etc